src/shared/components/palette/collection/write-collection-field.component.ts

```diff
@@ -43,8 +43,8 @@ export class WriteCollectionFieldComponent extends AbstractFieldWriteComponent i
       this.profileNotifier.profile.subscribe(_ => this.profile = _);
     }
     this.caseField.value = this.caseField.value || [];
-
     this.formArray = this.registerControl(new FormArray([]));
+    this.formArray.setErrors(null);
```

Comment:
please explain in details why this change

Reply:
DONE change reverted.